src/orchard/util/io.clj

```diff
@@ -0,0 +1,20 @@
+(ns orchard.util.io)
+
+(defn wrap-silently [f]
```

Comment:
I think this deserves a docstring. I think the name of the function is a bit confusing, but I can't suggest a better on right away. If it was a macro I'd say "without-output" or "suppress-output", but yeah - naming is hard.

Reply:
> I think this deserves a docstring.

Added!

maybe `wrap-with-silence`, IDK.

PR ready again

Reply:
I can live with the current name. :-)